### ISSUE TITLE
Can we publish a new release?

### ISSUE THREAD

**Issue by singhravi1** [aschn/drf-tracking#144]
Hi,

The last release was more than a year ago. Since then, there were many fixes and updates in the library.
Can we please release a new stable version?

**Comment by lingster** [comment]
@avelis : would it be possible to schedule a release? If you are snowed, I would gladly help as my fix #142 has not yet been released. Thanks in advance.

**Comment by avelis** [comment]
@lingster I have been having trouble finding time to continue pushing releases out. Any help would be greatly appreciated.

**Comment by lingster** [comment]
@avelis : more than happy to help. If you can set me up with (maintainer) rights to publish, I can take a look in the next day or so, I can also add some release notes to simplify deployment next time too.

**Comment by avelis** [comment]
@lingster I am actually not the owner and forgot to follow up and get ownership transfer years ago. You will have to reach out to @aschn about transfer of ownership or at least maintainer status.

**Comment by lingster** [comment]
@avelis on pypi you are setup as a maintainer of drf-tracking: https://pypi.org/project/drf-tracking/
I think you should be able to add me from there.

**Comment by lingster** [comment]
@aschn : could you kindly help to add me as maintainer to assist with the upkeep of this project?

**Comment by jdreben** [comment]
+1

**Comment by lingster** [comment]
@avelis : could you kindly permission me to deploy onto pypi? If not may I suggest that we fork this project+rename so that a new version could be published on pypi?

**Comment by avelis** [comment]
@lingster Yes. Just fork it and redirect people to it. I will even accept a README.md update to say that.

**Comment by lingster** [comment]
@avelis : ok so I have forked the repo and renamed as follows: https://github.com/lingster/drf-api-tracking. Currently in the process of getting all the CI working so it can be ready to accept PRs etc

**Comment by lingster** [comment]
@avelis : I have completed the fork and there is a new version 1.6.0 of drf-api-tracking which contains all changes so far. Could you kindly update the README and point people to the new repo?

**Comment by avelis** [comment]
@lingster Ok. Change made. Godspeed!
https://github.com/aschn/drf-tracking/blob/master/README.md

**Comment by avelis** [comment]
For anyone else who lands here [work as continued on in a fork now named drf-api-tracking](https://github.com/lingster/drf-api-tracking). 
Happy Tracking!

**Comment by lingster** [comment]
@avelis : thanks very much!
